plugins/nodes/src/js/components/NodesTable.js

```diff
@@ -64,11 +64,7 @@ var NodesTable = React.createClass({
     let requestReceived = this.props.receivedNodeHealthResponse;
 
     if (!requestReceived) {
-      return (
-        <Loader
-          innerClassName="loader-small"
-          type="ballBeat" />
-      );
+      return <Loader innerClassName="loader-small" type="ballBeat" />;
```

Comment:
Do we actually need the `innerClassName`? Would love to get rid of these props in favor of more descriptive props like `size="small"`...